src/main/java/com/hillel/items_exchange/config/MvcConfiguration.java

```diff
@@ -0,0 +1,37 @@
+package com.hillel.items_exchange.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.Ordered;
+import org.springframework.web.servlet.config.annotation.*;
+import org.springframework.web.servlet.view.JstlView;
+import org.springframework.web.servlet.view.UrlBasedViewResolver;
+
+
+@Configuration
+@EnableWebMvc
+@ComponentScan("com")
+public class MvcConfiguration implements WebMvcConfigurer{
```

Comment:
This class is excess - you should completely remove it.

Reply:
Performed.